setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -8,6 +8,14 @@ vs.localizedResources
                        title="Microsoft VisualStudio Managed Project System Common Files"
                        description="Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects"
 
+vs.dependencies
+  vs.dependency id=Microsoft.NetCore.Component.DevelopmentTools
```

Comment:
This dependency doesn't include us?

Reply:
No, with an asterisk that I haven't dug through ever layer of the graph exhaustively. You can see that package here: http://index/?query=%60Microsoft.NetCore.Component.DevelopmentTools&rightProject=VS(Repo)&file=src%5CSetupPackages%5CGroup%5CComponent%5CNetCore.DevelopmentTools%5Ccomponent.swr&line=3

I'm going to take this PR and generate an insertion that will hopefully cause the master package json file in the VS repo to be updated, and then there is a tool I can run that will output the complete closure of the dependency graph for a package, so intend to use that to verify.

Reply:
I'm looking at that and not understanding why we have a dependency on it? I think we'll get installed in situations that this isn't installed, such as desktop-only./

Reply:
To me this is part of what the user would expect when opening a project that uses our project system. eg. the other day you said "we may not depend directly on the debugger, but without it our project system is not very useful", well this package brings in the Microsoft.VisualStudio.PackageGroup.DotNetCoreDebugging package.

Reply:
Understood, but I don't this is that dependency, this depends on F# and .NET Core SDK which we do not depend on.

Reply:
I would expect the reverse dependency, in fact.